### PR TITLE
chore: support adaptive card to sync up templates

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: .github/scripts/pkg-version.sh [hunk 1]
```diff
@@ -32,4 +32,15 @@ elif [ $1 == 'core-template' ]; then
     echo "need to bump up templates' fallback version in fx-core"
     node ../.github/scripts/fxcore-sync-up-version.js
     git add ../packages/fx-core
+elif [ $1 == 'template-adaptive-card' ]; then
+    if [[ $SkipSyncup == *"template"* ]]; then
+        echo "skip sync up templates version with adaptive-card version"
+    elif [[ -z "$(git diff -- ../../templates)" ]]; then
+    echo "need bump up templates version since templates don not bump up by self"
+    node ../../.github/scripts/sdk-sync-up-version.js adaptivecards-tools-sdk yes;
```
Comment (wenytang-ms) [inline .github/scripts/pkg-version.sh:40]: this script will do sync up version to templates.
the sync up operation will update all the packages' dependencies of subitem.

File: packages/adaptivecards-tools-sdk/package.json [hunk 2]
```diff
@@ -12,7 +12,9 @@
     "lint:staged": "lint-staged",
     "check-sensitive": "npx eslint --plugin 'no-secrets' --cache --ignore-pattern 'package.json' --ignore-pattern 'package-lock.json'",
     "precommit": "npm run check-sensitive && lint-staged",
-    "format": "prettier --write --config ./.prettierrc.js --ignore-path ./.prettierignore \"src/**/*.ts\" \"*.{js,json}\""
+    "format": "prettier --write --config ./.prettierrc.js --ignore-path ./.prettierignore \"src/**/*.ts\" \"*.{js,json}\"",
+    "version": "../../.github/scripts/pkg-version.sh template-adaptive-card",
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/package.json:16]: do version sync up to templates.

File: packages/adaptivecards-tools-sdk/package.json [hunk 3]
```diff
@@ -12,7 +12,9 @@
...
+    "postversion": "../../.github/scripts/pkg-postversion.sh templates"
```
Comment (wenytang-ms) [inline packages/adaptivecards-tools-sdk/package.json:17]: add version tag to templates.

File: .github/scripts/pkg-version.sh [hunk 4]
```diff
@@ -32,4 +32,15 @@ elif [ $1 == 'core-template' ]; then
     echo "need to bump up templates' fallback version in fx-core"
     node ../.github/scripts/fxcore-sync-up-version.js
     git add ../packages/fx-core
+elif [ $1 == 'template-adaptive-card' ]; then
+    if [[ $SkipSyncup == *"template"* ]]; then
+        echo "skip sync up templates version with adaptive-card version"
+    elif [[ -z "$(git diff -- ../../templates)" ]]; then
+    echo "need bump up templates version since templates don not bump up by self"
```
Comment (IvanJobs) [inline .github/scripts/pkg-version.sh:39]: Nit: better to add indents to improve readability.

Comment (wenytang-ms) [reply .github/scripts/pkg-version.sh]: thanks!
Updated.

**Review by IvanJobs — APPROVED** [review]
lgtm

**Pull request merged**